data/themes/darktable.css

```diff
@@ -713,7 +713,7 @@ l
 
 #filter-box
 {
-  min-width: 50em;
+  min-width: 33em;
```

Comment:
Let's use 38 here. French labels tends to be longer than English one and I have some ellipsis for some entries in the sorting combo with 33. And even safer value could be 40.

Reply:
Done